packages/in_app_purchase/in_app_purchase_storekit/example/README.md

```diff
@@ -7,3 +7,75 @@ package.
 
 Unless you are making changes to this implementation package, this example is
 very unlikely to be relevant.
+
+# In App Purchase iOS Example
+
+### Preparation
+
+There's a significant amount of setup required for testing in app purchases
+successfully, including registering new app IDs and store entries to use for
+testing in App Store Connect. The App Store requires developers to configure
+an app with in-app items for purchase to call their in-app-purchase APIs.
+The App Store has extensive documentation on how to do this, and we've also
+included a high level guide below.
```

Comment:
Can we avoid adding a high level guide?  These things go out of date so quickly, and we don't do a good job of keeping track of when processes like this change. I'd rather just link to the Apple docs.  If we need something for our own reference for testing I'd rather it go into the wiki.

Reply:
This is a revert from https://github.com/flutter/packages/commit/bdb89758e11767c0e9770b46aee72ffe0301b3c1#diff-896f125f477201f602e98fef2399dfb4c17297b851fa7df48bd608a689f3c542

Without these instructions and with this plugin being maintained by a contract team but that contract ending I had zero idea how to test the example app. 

I can revert the ios changes but the android example app explicitly references the high level guide so android will keep its guide.